### PR TITLE
MAISTRA-1263 add replaces and olm.skipRange to csv files

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

**Comment by gbaufake** [comment]
@rcernich 
Will it be 1.0.9 replaced or 1.0.10?

**Comment by rcernich** [comment]
@gbaufake, it will, once 1.0.10 is released, or once the show up in the maistra-1.0 branch.

Which reminds me, I forgot to update the cni version for 1.0.

**Comment by rcernich** [comment]
Note, the changes to the servicemesh-operator.yaml file were made to minimize the differences between it and the maistra-operator.yaml file (so the only differences are the image names).

**Review by brian-avery — APPROVED** [review]

File: build/manifest-templates/clusterserviceversion.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ metadata:
     containerImage: __IMAGE_SRC__
     createdAt: __DATE__
     support: Red Hat, Inc. 
+    olm.skipRange: ">=1.0.2 <__VERSION__"
     alm-examples: |-
       [
         {
```
Comment (gbaufake) [inline build/manifest-templates/clusterserviceversion.yaml:20]: There are some missing commas on that JSON.
Lines: 23, 68 and 82

**Review by gbaufake — CHANGES_REQUESTED** [review]

File: build/manifest-templates/clusterserviceversion.yaml (hunk 1 above)
Comment (rcernich) [reply build/manifest-templates/clusterserviceversion.yaml:20]: fixed.

**Review by dgn — APPROVED** [review]

**Review by gbaufake — APPROVED** [review]

**Review by dmitri-d — APPROVED** [review]

**Pull request merged**